config.json

```diff
@@ -27,5 +27,11 @@
             "show": true,
             "pos": "bottom"
         }
+    },
+    "constant": {
+        "proxy_ip": "172.22.22.10",
+        "proxy_port": "3390",
```

Comment:
Since this is a public repo, please replace this with dummy values.
For example:
```
"proxy_ip": "xxx.xxx.xxx.xxx",
"proxy_port": "xxxxx",
```

Reply:
BTW, why do you write the port number in `str`, instead of `int`?

Reply:
There is no specific reason😅 Does it look better to set as `int`?

Reply:
😄 Yeah the port number is `int` in many cases.
Maybe later we might use some other ip/port, then it would use `int` for port numbers, so for consistency!